integrations/operator/crdgen/testdata/protofiles/teleport/legacy/client/proto/authservice.proto

```diff
@@ -285,6 +285,8 @@ message RouteToApp {
   string AzureIdentity = 6 [(gogoproto.jsontag) = "azure_identity,omitempty"];
   // GCPServiceAccount is the GCP service account to assume when accessing GCP API.
   string GCPServiceAccount = 7 [(gogoproto.jsontag) = "gcp_service_account,omitempty"];
+  // URI is the application URI.
+  string URI = 8 [(gogoproto.jsontag) = "uri,omitempty"];
```

Comment:
I don't think this change is necessary. This is just old test data for the operator.